spaghetti-core/src/main/java/com/prezi/spaghetti/obfuscation/internal/SymbolCollectVisitor.java

```diff
@@ -54,7 +54,7 @@
 
 	@Override
 	public Collection<String> visitConstNode(ConstNode node) {
-		return extractNames(node.getEntries());
+		return ImmutableList.<String>builder().add(node.getName()).addAll(extractNames(node.getEntries())).build();
```

Comment:
Right, but why not do it in one step? Like:

```groovy
  private static Collection<String> extractNames(EnumNode enumNode) {
    ImmutableList.Builder<String> builder = ImmutableList.builder();
    builder.add(enumNode.getName());
    for (NamedNode node : enumNode.getValues()) {
      builder.add(node.getName();
    }
    return builder.build();
  }
```

Probably not such a big concern performance-wise, but still.